src/Microsoft.HttpRepl/Commands/BaseHttpCommand.cs

```diff
@@ -37,6 +37,12 @@ public abstract class BaseHttpCommand : CommandWithStructuredInputBase<HttpState
         private const string StreamingOption = nameof(StreamingOption);
         private const string BodyContentOption = nameof(BodyContentOption);
         private static readonly char[] HeaderSeparatorChars = new[] { '=', ':' };
+        private static readonly Dictionary<string, string> FileExtensionLookup = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
+        {
+            { "application/json", ".json" },
```

Comment:
technically, text/json also exists, though it's not preferred or as widespread.

Reply:
added.